docs/how-to-guides/export-data.md

```diff
@@ -0,0 +1,124 @@
+| `Identifier` | The identifier for the DICOM study, series, or SOP instance in the format of `"<study instance UID>[/<series instance UID>[/<SOP instance UID>]]"` |
+| `Error`      | The error message |
+
+## Sources
+
+While the API can support different sources, only `"identifiers"` are supported today.
+
+### Identifiers
+
+**Type**: `"identifiers"`
+
+The identifiers source can be used to specify a list of DICOM studies, series, and/or SOP instances for export.
+
+#### Settings
+
+The only setting is the list of identifiers to export.
+
+| Property | Required | Default | Description |
+| :------- | :------- | :------ | :---------- |
+| `Values` | Yes      |         | A list of one or more DICOM studies, series, and/or SOP instances identifiers in the format of `"<study instance UID>[/<series instance UID>[/<SOP instance UID>]]"` |
+
+#### Example
+
+```json
+{
+    "type": "identifiers",
+    "settings": {
+        "values": [
+            "1.2.3",
+            "12.3/4.5.678",
+            "123.456/7.8/9.1011.12"
+        ]
+    }
+}
+```
+
+## Destinations
+
+While the API can support different destinations, only `"azureblob"` is supported today.
+
+### Azure Blob
+
+**Type**: `"azureblob"`
+
+The Azure Blob storage destination can be used to export DICOM files to an existing Azure Blob container.
+
+#### Settings
+
+The connection to the Azure Blob storage account can be specified with either a `ConnectionString` and `BlobContainerName` or a `BlobContainerUri`. One of these settings is required!
+
+If the storage account requires authentication, a [SAS token](https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview) can be included in either the `ConnectionString` or `BlobContaienrUri`. A managed identity can also be used to access the storage account with the `UseManagedIdentity` option. Note that the identity used by both the DICOM server and the functions must have access to the container.
+
+| Property             | Required | Default | Description |
+| :------------------- | :------- | :------ | :---------- |
+| `BlobContainerName`  | No       | `""`    | The name of a blob container. Only required when `ConnectionString` is specified |
+| `BlobContainerUri`   | No       | `""`    | The complete URI for the blob container                      |
+| `ConnectionString`   | No       | `""`    | The [Azure Storage connection string](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string) that must minimally include information for blob storage |
+| `UseManagedIdentity` | No       | `false` | An optional flag indicating whether managed identity should be used to authenticate to the blob container |
+
+#### Error Log
+
+The error log can be found at `<export blob container uri>/<operation ID>/Errors.log`.
+
+### Example
+
+```json
+{
+    "type": "azureblob",
+    "settings": {
+        "blobContainerName": "export",
+        "connectionString": "UseDevelopmentStorage=true"
```

Comment:
Is this correct?  It says "connectionString" but looks like a setting 🤔

Reply:
This is indeed correct lol. The connection strings for Azure Storage are a series key-value pairs. For the emulator, it just needs this value